api/src/pcapi/sandboxes/scripts/creators/industrial/create_industrial_incidents.py

```diff
@@ -1,18 +1,261 @@
+    )
+    if is_partial and not multiple_bookings:
+        new_total_amount = (incident_bookings[0].amount * incident_bookings[0].quantity) * 100 - 1000
+    else:
+        new_total_amount = 0
+
+    for booking in incident_bookings:
+        finance_factories.IndividualBookingFinanceIncidentFactory(
+            booking=booking, incident=finance_incident, newTotalAmount=new_total_amount
+        )
+
+
+def _create_one_collective_incident(
+    offerer: offerers_models.Offerer,
+    iteration: int,
+    with_other_venue: bool,
+    with_over_revenue_threshold_booking: bool,
+    multiple_bookings: bool,
+    force_debit_note: bool,
+) -> None:
+    reimbursement_point_venue = offerers_factories.VenueFactory(
+        name=f"{iteration} - Point de remboursement avec beaucoup d'incidents collectifs",
+        managingOfferer=offerer,
+        pricing_point="self",
+        reimbursement_point="self",
+    )
+    finance_factories.BankInformationFactory(venue=reimbursement_point_venue)
+    if with_other_venue:
+        other_venue = offerers_factories.VenueFactory(
+            name=f"{iteration} - Autre lieu avec beaucoup d'incidents collectifs",
+            managingOfferer=offerer,
+            pricing_point=reimbursement_point_venue,
+            reimbursement_point=reimbursement_point_venue,
+        )
+
+    if with_over_revenue_threshold_booking:
+        # This is a quick way to have a Venue reach the revenue threshold to reach the next ReimbursementRule
+        special_booking = _add_nearly_over_revenue_threshold_booking(reimbursement_point_venue, True)
+
+    stocks = educational_factories.CollectiveStockFactory.create_batch(
+        2 if multiple_bookings else 1,
+        collectiveOffer__venue=other_venue if with_other_venue else reimbursement_point_venue,
+        price=30,
+    )
+    incident_bookings = []
+    for stock in stocks:
+        booking = educational_factories.UsedCollectiveBookingFactory(collectiveStock=stock)
+        incident_bookings.append(booking)
+
+    bookings = ([special_booking] if with_over_revenue_threshold_booking else []) + incident_bookings
+    for booking in bookings:
+        finance_factories.UsedCollectiveBookingFinanceEventFactory(collectiveBooking=booking)
+    for booking in bookings:
+        event = finance_models.FinanceEvent.query.filter_by(collectiveBooking=booking).one()
+        finance_api.price_event(event)
+    finance_api.generate_cashflows_and_payment_files(cutoff=datetime.datetime.utcnow())
+
+    finance_incident = finance_factories.FinanceIncidentFactory(
+        venue=other_venue if with_other_venue else reimbursement_point_venue, forceDebitNote=force_debit_note
+    )
+    new_total_amount = 0
+
+    for booking in incident_bookings:
+        finance_factories.CollectiveBookingFinanceIncidentFactory(
+            collectiveBooking=booking, incident=finance_incident, newTotalAmount=new_total_amount
```

Comment:
Détail : Plus simplement en enlevant la variable au-dessus :
```suggestion
            collectiveBooking=booking, incident=finance_incident, newTotalAmount=0
```